src/documents/forms/EditDocumentForm.js

```diff
@@ -141,22 +156,81 @@ const EditDocumentForm = React.forwardRef(
                     </Form.Field>
                   ))}
                 </Form.Field>
-                <Form.Field required>
-                  <label>Describe Document Contents:</label>
-                  <TextArea
-                    className={
-                      touched.file_desc && errors.file_desc
-                        ? 'border-red'
-                        : null
-                    }
-                    data-testid="description-input"
-                    type="text"
-                    name="file_desc"
-                    value={values.file_desc}
-                    onChange={handleChange}
-                    onBlur={handleBlur}
-                  />
-                </Form.Field>
+                {submitButtons && (
+                  <Form.Field required>
+                    <label>Describe Document Contents:</label>
+                    <Segment
+                      className={
+                        touched.file_desc && errors.file_desc
+                          ? 'border-red'
+                          : null
+                      }
+                    >
+                      <Editor
```

Comment:
Is it worth extracting this to its own component so we don't need to redefine the settings everywhere?